go/test/endtoend/cluster/cluster_util.go

```diff
@@ -95,16 +95,21 @@ func GetPrimaryPosition(t *testing.T, vttablet Vttablet, hostname string) (strin
 // This is used to check that replication has caught up with the changes on primary.
 func VerifyRowsInTabletForTable(t *testing.T, vttablet *Vttablet, ksName string, expectedRows int, tableName string) {
 	timeout := time.Now().Add(1 * time.Minute)
+	lastNumRowsFound := 0
 	for time.Now().Before(timeout) {
 		// ignoring the error check, if the newly created table is not replicated, then there might be error and we should ignore it
 		// but eventually it will catch up and if not caught up in required time, testcase will fail
 		qr, _ := vttablet.VttabletProcess.QueryTablet("select * from "+tableName, ksName, true)
 		if qr != nil && len(qr.Rows) == expectedRows {
 			return
 		}
+		if qr != nil {
+			lastNumRowsFound = len(qr.Rows)
+		}
 		time.Sleep(300 * time.Millisecond)
 	}
-	assert.Fail(t, "expected rows not found.")
+	s := fmt.Sprintf("Want %d rows, got %d, in %s (%s.%s)", expectedRows, lastNumRowsFound, vttablet.Alias, ksName, tableName)
+	assert.Fail(t, s)
```

Comment:
Please use this paradigm:
```suggestion
	require.Equalf(t, expectedRows, lastNumRowsFound, "unexpected number of rows in %s (%s.%s)", vttablet.Alias, ksName, tableName)
```

"want" and "got" is implicitly reported, in well tabulated format, by `require/assert`.

Reply:
done